src/Strategy/Erlang/Rebar3Tree.hs

```diff
@@ -40,8 +44,37 @@ rebar3TreeCmd = Command
   , cmdAllowErr = Never
   }
 
-analyze :: (Has Exec sig m, Has Diagnostics sig m) => Path Abs Dir -> m ProjectClosureBody
-analyze dir = mkProjectClosure dir <$> execParser rebar3TreeParser dir rebar3TreeCmd
+configFile :: Path Rel File
+configFile = $(mkRelFile "rebar.config")
+
+analyze :: (Has Exec sig m, Has ReadFS sig m, Has Diagnostics sig m) => Path Abs Dir -> m ProjectClosureBody
+analyze dir = do
+  aliasMap <- extractAliasLookup <$> readContentsParser parseConfig (dir </> configFile)
+  mkProjectClosure dir . unaliasDeps aliasMap <$> execParser rebar3TreeParser dir rebar3TreeCmd
+
+extractAliasLookup :: ConfigValues -> M.Map Text Text
+extractAliasLookup (ConfigValues erls) = foldr extract M.empty erls
+  where
+    extract :: ErlValue -> M.Map Text Text -> M.Map Text Text
+    extract val aliasMap = aliasMap <> M.fromList (mapMaybe getAlias packages)
+      where
+        packages :: [ErlValue]
+        packages = case val of
+          ErlTuple [ErlAtom (AtomText "deps"), ErlArray deplist] -> deplist
+          _ -> []
+        
+        getAlias :: ErlValue -> Maybe (Text, Text)
+        getAlias erl = case erl of
+          ErlTuple [ErlAtom (AtomText realname), ErlString _, ErlTuple [ErlAtom (AtomText "pkg"), ErlAtom (AtomText alias)]] -> Just (realname, alias)
```

Comment:
This pattern match is a beast, but it actually seems like the most idiomatic thing.  It's kind of a weird structure to find, functionally, so expressing it as-is seems to make sense.